packages/tooltip/src/vaadin-tooltip-mixin.js

```diff
@@ -617,6 +617,13 @@ export const TooltipMixin = (superClass) =>
       }
     }
 
+    /** @protected */
+    __onOverlayClick(event) {
+      // Prevent click listeners from being called when
+      // the tooltip is slotted into the target element
+      event.stopPropagation();
```

Comment:
The button still triggers its click animation when clicking the tooltip. Maybe it would be good to fix that as well.

Reply:
Thanks, fixed. Changed to stop propagation for `mousedown` too - it's a better fix than the previous one. The active state used by click animation is set on `mousedown` event in `ActiveMixin` so stopping it makes sense.